### PR TITLE
Add removable device attribute to disk object

### PR DESCRIPTION
**Pull request by thetravischannel**
I love your commitment to this project, and I am definitely getting great use out of it!

This PR is just to add a bool indicating whether a disk is a removable device or not.

Thanks again!

### CODE REVIEW

File: block.go [hunk 1]
```diff
@@ -89,6 +89,7 @@ type Disk struct {
 	SizeBytes              uint64            `json:"size_bytes"`
 	PhysicalBlockSizeBytes uint64            `json:"physical_block_size_bytes"`
 	DriveType              DriveType         `json:"drive_type"`
+	Removable              bool              `json:"removable"`
```
Comment (jaypipes) [inline block.go]: How about naming this `IsRemovable` to match the `IsReadOnly` boolean attribute of `Partition`?

File: block_linux.go [hunk 2]
```diff
@@ -310,6 +310,19 @@ func (ctx *context) diskPartitions(disk string) []*Partition {
 	return out
 }
 
+func (ctx *context) isRemovable(disk string) bool {
```
Comment (jaypipes) [inline block_linux.go]: suggest naming the method `diskIsRemovable()` just to match the other methods like `diskPartitions()` or `diskIsRotational()`

File: block_darwin.go [hunk 3]
```diff
@@ -259,6 +259,7 @@ func (ctx *context) blockFillInfo(info *BlockInfo) error {
 			SizeBytes:              uint64(disk.Size),
 			PhysicalBlockSizeBytes: uint64(infoPlist.DeviceBlockSize),
 			DriveType:              ctx.driveTypeFromPlist(infoPlist),
+			Removable:              infoPlist.Removable,
```
Comment (jaypipes) [inline block_darwin.go]: `IsRemovable` :)

Comment (thetravischannel) [reply block_darwin.go]: Aw man, my bad. I knew I probably missed something.

**Pull request merged**